cmd/pipeline/deploy.go

```diff
@@ -134,6 +136,7 @@ okteto pipeline deploy --wait=false`,
 	}
 
 	cmd.Flags().StringVarP(&flags.name, "name", "p", "", "name of the pipeline (defaults to the git config name)")
+	cmd.Flags().StringVarP(&flags.k8sContext, "context", "c", "", "context where the development environment was deployed")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&flags.k8sContext, "context", "c", "", "context used to deploy the pipeline")
```

Reply:
all the other context flags have the same copy, before changing only this one we should ask product.